src/components/create-vm-dialog/createVmDialogUtils.js

```diff
@@ -145,3 +146,11 @@ export function correctSpecialCases(os) {
 export function needsRHToken(osName) {
     return osName.startsWith("rhel");
 }
+
+export function saveOfflineToken(offlineToken) {
+    return cockpit.user().then(user => localStorage.setItem(user.id + "-offline-token", offlineToken));
+}
+
+export function loadOfflineToken(setToken) {
+    return cockpit.user().then(user => setToken(localStorage.getItem(user.id + "-offline-token")));
```

Comment:
This needs a test. AFAIK the current tests already create a fake server and token, thus this functionality should be testable (log out and back in)